src/usr/local/www/services_pppoe_edit.php

```diff
@@ -237,12 +237,12 @@ function vpn_pppoe_get_id() {
 		$users = array();
 		for ($x = 0; $x < 4999; $x++) {
 			if ($_POST["username{$x}"]) {
-				$usernam = $_POST["username{$x}"] . ":" . base64_encode($_POST["password{$x}"]);
+				$username = $_POST["username{$x}"] . ":" . base64_encode($_POST["password{$x}"]);
 				if ($_POST["ip{$x}"]) {
-					$usernam .= ":" . $_POST["ip{$x}"];
+					$username .= ":" . $_POST["ip{$x}"];
 				}
 
-				$users[] = $usernam;
+				$users[] = $username;
```

Comment:
This should be harmless.

Reply:
No conflict here, variable rename is fine.